src/avfoundation.cs

```diff
@@ -11229,7 +11229,8 @@ interface AVPlayerItem_AVPlaybackRestrictions {
 		void CancelPlaybackRestrictionsAuthorizationRequest ();
 	}
 	
-	[iOS (13,0)][Watch (6,0)]
+	// trick the generator to believe it is present in an earlier version of the OS.
```

Comment:
It seems that the type was already present
https://developer.apple.com/documentation/avfoundation/avplayeritemoutput?language=objc
and marking it as `[iOS (13,0)]` is what cause the crash on 32 bits
IOW the comment is wrong

but if it has been correct (new base types do happen) then all members would need to have updated availability attributes,